views/_includes/backButton.pug

```diff
@@ -1,2 +1,2 @@
 if prevRoute
-  a(class='back-button' href=prevRoute.path) <span aria-hidden="true">← </span> Go back
\ No newline at end of file
+  a(class='back-button' href=prevRoute.path) <span aria-hidden="true">◀ </span> Go back
```

Comment:
- the arrow could be applied with CSS (using `::before`) but since you have `aria-hidden="true"`; it's probably fine
- "Go back" should be translated. So like `#{__('Go back')}`. 

If you run the app in dev mode and then switch between english and french, it should should up in "/locales" for en.json and fr.json.